### DISCUSSION TITLE
the pics of enlarged are always  messes after stablesr operated...

### DISCUSSION THREAD

**Discussion by lsh1611** [pkuliyi2015/sd-webui-stablesr#63] [Q&A]
Hello, 
I followed the steps from here: https://github.com/pkuliyi2015/sd-webui-stablesr/blob/master/README_CN.md
but it's still failed:
![1](https://github.com/pkuliyi2015/sd-webui-stablesr/assets/17892859/1db17fa8-bd9b-404b-b479-912e1bdb42a8)
![2](https://github.com/pkuliyi2015/sd-webui-stablesr/assets/17892859/5f2eaf86-31fd-431a-8f37-73adf8b6290d)
![3](https://github.com/pkuliyi2015/sd-webui-stablesr/assets/17892859/26cb7817-0cb7-4b9f-acd0-d4f2d305e606)

the pic is mess...
Could anyone tell me the key issue of this.

**Comment by 100640002** [comment]
Many people have this problem now。
